lib/fsm/kubernetes.go

```diff
@@ -104,6 +105,21 @@ func GetUpsertBootstrapResourceFunc(client *kubernetes.Clientset) resources.Reso
 				return trace.Wrap(rigging.ConvertError(err))
 			}
 			log.Debugf("Updated PodSecurityPolicy %q.", resource.Name)
+		case *v1beta1.PodSecurityPolicy:
```

Comment:
This has been reverted to support RBAC configurations from older versions when using intermediate hops.